inspirehep/modules/orcid/receivers.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from flask import current_app
+
+from invenio_oauthclient.models import RemoteAccount
+
+from invenio_records.signals import before_record_insert, before_record_update
+
+import orcid
+
+from .utils import convert_to_orcid
+
+
+@before_record_insert.connect
+@before_record_update.connect
+def send_records_to_orcid(sender, *args, **kwargs):
+    """Sends updated records to orcid."""
+    try:
+        orcid_json = convert_to_orcid(sender)
+        api = orcid.MemberAPI(current_app.config['OAUTHCLIENT_ORCID_CREDENTIALS']['consumer_secret'],
+                              current_app.config['OAUTHCLIENT_ORCID_CREDENTIALS'][
+                                  'consumer_key'],
+                              sandbox=True
```

Comment:
Well the flag is still useful, (e.g. we can `qa.inspirehep.net` or `localhost` still using the sandbox).